src/commands/fun.command.ts

```diff
@@ -0,0 +1,344 @@
+            });
+        }
+    }
+
+    @Command("feed")
+    @Description("Feed another user. Must @ another user")
+    @Guard(NotBot, Mention)
+    async feed(command: CommandMessage): Promise<void> {
+        this.logger.info("Sending feed");
+
+        const member = await command.guild.member(command.author);
+        const author = member ? member.nickname : command.author.username;
+        const user = getMentioned(command);
+
+        const feed = await (await this.neko.sfw.feed()).url;
+
+        const embed = new MessageEmbed()
+            .setColor(COLOR.RANDOM)
+            .setTitle(`${author} fed ${user}`)
+            .setImage(feed)
+            .setFooter("You fed them good food!");
+
+        if (user === "error") { return; }
+        else {
+
+            command.reply(embed).then((messageSent) => {
+                this.logger.info(`Sent a feed : message id ${messageSent.id}`);
+            });
+        }
+    }
+
+    @Command("ping")
+    @Description("Get latency of the bot")
+    @Guard(NotBot)
+    async ping(command: CommandMessage): Promise<void> {
+        this.logger.info("Sending feed");
+        const time = Date.now() - command.createdTimestamp;
+        command.reply("Pong time is " + time).then((messageSent) => {
+            this.logger.info(`Sent Ping : message id ${messageSent.id}`);
+        });
+    }
+
+    @Command("dice")
+    @Description("Roll a dice")
+    @Guard(NotBot)
+    async dice(command: CommandMessage): Promise<void> {
+        this.logger.info("Sending dice");
+        const diceRoll = Math.floor(Math.random() * 6) + 1;
+
+        command.reply("You rolled a " + diceRoll).then((messageSent) => {
+            this.logger.info(`Sent dice : message id ${messageSent.id}`);
+        });
+    }
+
+    @Command("magic8")
+    @Description("Get what the magic 8 ball says")
+    @Guard(NotBot)
+    async magic8(command: CommandMessage): Promise<void> {
+        this.logger.info("Sending magic8");
+        const options = ["It is certain.", "It is decidedly so.", "Without a doubt.", "Yes - definitely.", "Most likely.", "Outlook good.", "Signs point to yes.", "Reply hazy, try again.", "Ask again later.",
+            "Better not tell you now.", "Cannot predict now.", "Concentrate and ask again.", "Don't count on it.", "My reply is no.", "My sources say no.", "Outlook not so good.", "Very doubtful."];
+
+        const random = Math.floor(Math.random() * options.length);
+
+
+        command.reply("The magic8 ball says " + options[random]).then((messageSent) => {
+            this.logger.info(`Sent magic8 : message id ${messageSent.id}`);
+        });
+    }
+
+    @Command("kill")
+    @Description("Kill another user. Uh Oh")
+    @Guard(NotBot, Mention)
+    async kill(command: CommandMessage): Promise<void> {
+        this.logger.info("Sending kill");
+        const options = ["barbecued", "disintegrated", "360-no-scoped",
```

Comment:
Noticed that these might make it a bit hard, what you could do is keep the function decorators and pass these in as arguments into a generic function (along with the command) rather than duplicating the code for making the reply.